### ISSUE TITLE
h1 title with <code> element not recognized

### ISSUE THREAD

**Issue by BryceRussell** [CloudCannon/pagefind#181]
Some of my documentation pages have back ticks in the `h1` title like 
```
# `<Component>`
```
and the title does not get recognized for search results

**Comment by bglw** [comment]
Ah, good find, this actually helps track down #67 — The issue is that the `<Component>` text isn't being correctly escaped by the Pagefind UI, so the browser is actually rendering it as `<component></component>`. I'll get this fixed up.

**Comment by bglw** [comment]
For a quick fix, you could add a result hook to Pagefind:

```js
new PagefindUI({
    element: "#search",
    processResult: function (result) {
        result.meta.title = result.meta.title.replace(/</g, "&lt;").replace(/>/g, "&gt;")
        return result;
    }
});
```

**Comment by BryceRussell** [comment]
It works! Thanks for the quick response, I was also having the same problems mentioned in #67 with some of my code blocks getting turned to HTML

**Comment by bglw** [comment]
Hey @BryceRussell  ! 👋

Good news — this is fixed in Pagefind v1.0.0! ✨

See the full release notes here: https://github.com/CloudCannon/pagefind/releases/tag/v1.0.0 💙
